src/site/pages/accessibility-guide.md

```diff
@@ -141,14 +141,18 @@ The [WCAG 2.0 techniques](https://www.w3.org/TR/WCAG20-TECHS/Overview.html#conte
 
 
 ## Testing
-To keep testing simple and easy to complete, we ask that contributors complete the following tests to try to catch most of the accessibility issues that may be present:
-1. Test keyboard accessibility, and check that these requirements are met:
+Many accessibility issues can be found by doing a few simple checks:
+1. Use an accessibility audit tool to check for violations. If you are using PatternFly in your project, we recommend using [aXe: The Accessibility Engine](https://www.deque.com/axe/) to check for accessibility violations. If you are contributing to PatternFly, refer to our [README.md](https://github.com/patternfly/patternfly-next/blob/master/README.md) on how to run this tool.
```

Comment:
should the `README.md` link go to the "testing for a11y" anchor? https://github.com/jgiardino/patternfly-next/tree/a11y-testing#testing-for-accessibility